src/ucs/debug/memtrack.c

```diff
@@ -203,7 +202,17 @@ void *ucs_realloc(void *ptr, size_t size, const char *name)
 
 void *ucs_memalign(size_t boundary, size_t size, const char *name)
 {
+
+#if defined(HAVE_MEMALIGN)
```

Comment:
@kostikbel 
in memtrack.h, in release builds where memtrack is disabled - ucs_memalign is defined by a macro to be memalign, so it would not work on freebsd.
i would suggest to instead remove all usages/references of memalign in the code, remove it from memtrack (along with include malloc.h), and switch to using posix_memalign. WDYT?